prestoadmin/__init__.py

```diff
@@ -17,30 +17,89 @@
 
 __version__ = '1.2'  # Make sure to update setup.py too
 
+
 import os
+import sys
+
+main_dir = os.path.dirname(os.path.abspath(os.path.dirname(__file__)))
 
 from fabric.api import env
 
+import fabric_patches
+import mode
+
+from prestoadmin.mode import get_mode, for_mode, MODE_STANDALONE, MODE_SLIDER
+from prestoadmin.util.exception import ConfigFileNotFoundError, \
+    ConfigurationError
+
+
+#
+# Subcommands common to all modes. If anybody knows why fabric_patches is in
+# the list, I'll make a note for the next person.
+#
+__all__ = ['fabric_patches', 'mode']
+
+cfg_mode = None
+try:
+    cfg_mode = get_mode()
```

Comment:
I think defaulting to standalone would be good, that way we don't change how it behaves for current presto-admin users.